### DISCUSSION TITLE
Daikin Altherma 3 H HT Heat Pump

### DISCUSSION THREAD

**Discussion by matthewgunners** [revk/xESP32-Faikout#269] [Q&A]
Hi, is the Faikin controller compatible with the Daikin Altherma 3 H HT Heat Pump, and if so is there any changes that need to be made to the code?

**Comment by revk** [comment]
I think that is covered by https://github.com/revk/ESP32-Faikin/discussions/248#discussioncomment-8850015

We think it may be read only, but I hope to have that in the Faikin anyway.
